cmd/gaia/cli_test/test_helpers.go

```diff
@@ -60,13 +60,16 @@ var (
 
 // Fixtures is used to setup the testing environment
 type Fixtures struct {
-	ChainID  string
-	RPCAddr  string
-	Port     string
-	GDHome   string
-	GCLIHome string
-	P2PAddr  string
-	T        *testing.T
+	BuildDir   string
+	GDBinary   string
```

Comment:
Can we rename all these to `GaiadBinary`, `GaiacliBinary `, etc. or at least add a comment? Just by reading the property I didn't get what was it referring to. I had to go to the `New...` function to understand it

Reply:
Done